### PR TITLE
22549 Misc updates for change-requested continuation application

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#22549

*Description of changes:*
- app version = 5.11.10
- enable validation right away if filing is Change Requested
- added label props to effective data time component
- updated continuation authorization layout per designs
- now manage authorization files validity separately
- passed in custom values for radio button labels
- updated unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1364,6 +1365,11 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     ) {
       this.setShowErrors(true)
     }
+
+    // enable validation right away if user is changing a submitted filing
+    if (this.getFilingStatus === FilingStatus.CHANGE_REQUESTED) {
+      this.setShowErrors(true)
+    }
```
Comment (severinbeauvais) [inline src/App.vue:1372]: In this scenario, the application should be fully valid, except we're going to make the Continuation Authorization files section invalid to bring the user's attention to the fact they need to make changes to their file(s) and we want to bring their attention to this. Ie,

image

File: src/components/ContinuationIn/ContinuationAuthorization.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/ContinuationAuthorization.vue:1]: For the most part, the template (HTML) in here was just moved around.

Also this content is now broken up in to 2 v-cards. (Previously, this sub-component was wrapped in 1 v-card in the parent component.) This is to show validation separately on the 2 v-cards.

File: src/components/ContinuationIn/ContinuationAuthorization.vue [hunk 2]
```diff
@@ -296,30 +352,23 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
...
 }
 </script>
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
-header {
-  p, li {
-    color: $gray7;
-  }
```
Comment (severinbeauvais) [inline src/components/ContinuationIn/ContinuationAuthorization.vue:322]: I got rid of the header tag in here.

File: src/views/ContinuationIn/ContinuationInReviewConfirm.vue
Comment (severinbeauvais) [inline src/views/ContinuationIn/ContinuationInReviewConfirm.vue:1]: image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-723-4ofwptq9.web.app

**Review by ArwenQin — APPROVED** [review]
Looks good!

**Pull request merged**